src/components/Editor/SearchBar.js

```diff
@@ -399,12 +402,15 @@ const SearchBar = React.createClass({
   },
 
   // Handlers
-  selectResultItem(item: FormattedSymbolDeclaration) {
+  selectResultItem(e: SyntheticEvent, item: SymbolDeclaration, index?: number) {
```

Comment:
For _line length_ issue we can always wrap arguments to different lines like https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/ConditionalPanel.js#L10

I wonder why we have `index` function parameter. It doesn't appear anywhere in function.